cmd/ndm_daemonset/probe/mountprobe.go

```diff
@@ -61,10 +68,14 @@ var mountProbeRegister = func() {
 		}
 	}
 	newRegisterProbe := &registerProbe{
-		priority:   mountProbePriority,
-		name:       mountProbeName,
-		state:      mountProbeState,
-		pi:         &mountProbe{Controller: ctrl},
+		priority: mountProbePriority,
+		name:     mountProbeName,
+		state:    mountProbeState,
+		pi: &mountProbe{
```

Comment:
Can you do a small refactoring here, similar like in [udeprobe](https://github.com/openebs/node-disk-manager/blob/master/cmd/ndm_daemonset/probe/udevprobe.go#L82-L116) to make sure we maintain the same pattern.